### PR TITLE
ci: deploy preview to vercel

### PR DESCRIPTION
**Pull request by hexf00**

### CODE REVIEW

**Comment by hexf00** [comment]
@jikkai Can you review this? Thx.

**Comment by jikkai** [comment]
> @jikkai Can you review this? Thx.

Some checks were not successful :(

File: .github/workflows/preview-deploy.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+name: 📤 Preview Deploy
+on:
+  pull_request:
+    branches:
+      - main
+  push:
```
Comment (jikkai) [inline .github/workflows/preview-deploy.yml:6]: There is no need to build the preview link when pushing to the `main` branch, this is what other workflows do

Comment (hexf00) [reply .github/workflows/preview-deploy.yml]: I removed it. You made sense.

**Comment by hexf00** [comment]
> > @jikkai Can you review this? Thx.
> 
> Some checks were not successful :(

Now it looks good.

**Review by jikkai — APPROVED** [review]

**Pull request merged**